### DISCUSSION TITLE
Public information display

### DISCUSSION THREAD

**Discussion by JanKoppe** [pretalx/pretalx#1068] [Ideas]
## Problem you are facing

See #790, which was closed in favor of #197 . The remaining part was some solution for a "quick glance" information display, both for the speaker (how much time do I have remaining?) and the tech personal (upcoming talks, preparations necessary?)

## Possible Solution

A few additional views that will show
* a countdown view for the current talk that does not distract the speaker, but gives a clear indication when time is running out (e.g. <5 minutes remaining background goes from black to red)
* some detailed calendar-ish view for tech personal, showing the immediately coming up talks with speaker information, maybe contact detail, can be extended with the suggested notes, etc from #197.

**Comment by Gailogail** [comment]
A view which shows a countdown timer to the start and finish of a session would be very useful for session chairs, speakers and audiences. It has the benefit of giving full transparency of the timer to all present, as well as enabling chairs in multiple tracks to be fully synchronised (and hopefully strict, since there is no room for ambiguity and everyone can see this)
